production/scrapers/utah_statewide.R

```diff
@@ -0,0 +1,116 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+utah_statewide_check_date <- function(x, date = Sys.Date()){
+    z <- xml2::read_html(x)
+    
+    z %>%
+        rvest::html_node(
+            xpath="/html/body/section/article/div[2]/div[3]/p[24]/em/span") %>%
+        rvest::html_text() %>%
+        str_remove("(?i)updated") %>%
+        lubridate::mdy() %>%
+        error_on_date(date)
+}
+
+utah_statewide_pull <- function(x){
+    page_html <- xml2::read_html(x)
+    return(page_html)
+}
+
+utah_statewide_restruct <- function(x){
+    staff_confirmed <- rvest::html_nodes(x, 
+                                         xpath = "/html/body/section/article/div[2]/div[3]/p[22]/span/em/strong/span") %>%
+        rvest::html_text(.) 
+    if(!str_detect(staff_confirmed, "(?i)total confirmed staff cases")){
+        warning("Field does mot match expected text")
+    }
+    
+    staff_recovered <- rvest::html_nodes(x,
+                                         xpath = "/html/body/section/article/div[2]/div[3]/p[23]/strong/span") %>%
+        rvest::html_text()
+    if(!str_detect(staff_recovered, "(?i)total recovered staff cases")){
+        warning("Field does mot match expected text")
+    }
+    restruct_out <- list(staff_confirmed = staff_confirmed,
+                         staff_recovered = staff_recovered)
+    return(restruct_out)
+}
+
+utah_statewide_extract <- function(x){
+    statewide_staff_confirmed <- x$staff_confirmed %>%
+        str_split("(?i)Total Confirmed Staff Cases: ") %>%
+        unlist() %>%
+        .[2] %>%
+        as.numeric() 
+    statewide_staff_recovered <- x$staff_recovered %>%
+        str_split("(?i)Total Recovered Staff Cases: ") %>%
+        unlist() %>%
+        .[2] %>%
+        as.numeric() 
+    
+    staff_rows <- tibble(Name = "STATEWIDE",
+                         Staff.Confirmed = statewide_staff_confirmed,
+                         Staff.Recovered = statewide_staff_recovered,
+                         Staff.Active = statewide_staff_confirmed - statewide_staff_recovered)
```

Comment:
Technically, `Staff.Confirmed - Staff.Recovered = Staff.Active + Staff.Deaths`, so fully attributing that difference to `Staff.Active` may not be accurate. It doesn't seem like Utah reports data on staff deaths, so we don't actually know though. 

My personal vote is to remove this line (especially since we can always `diff_roll_sum` estimate), but open to being overruled!

Reply:
Good point, I spaced out about staff deaths in this calculation! Just pushed a change where I removed that line.